src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
```

Comment:
please add some header comments for the class to tell what it is supposed to do.